src/utils/src/net/mac.rs

```diff
@@ -60,30 +62,31 @@ impl MacAddr {
     /// # Example
     ///
     /// ```
+    /// use std::str::FromStr;
+    ///
     /// use self::utils::net::mac::MacAddr;
-    /// MacAddr::parse_str("12:34:56:78:9a:BC").unwrap();
+    /// MacAddr::from_str("12:34:56:78:9a:BC").unwrap();
```

Comment:
does using from_str instead of parse_str improve logging or is it done just to use standard implementation?

Reply:
Just done to use the standard implementation.